apps/nextjs/src/components/navbar/navbar-right-menu.tsx

```
@@ -22,18 +26,24 @@ export default async function RightMenu() {
+        >
+          <SolarMagniferLinear className={sharedClassNames} />
+        </Button>
+      </Link>
       </Button>
 
       <Link prefetch={true} passHref href="/dashboard?tab=settings">
-        <Button className="p-1">
-          <Icon className={sharedClassNames} icon="solar:settings-linear" />
+        <Button className="flex items-center justify-center bg-transparent p-1">
+          <SolarSettingsLinear className={sharedClassNames} />
         </Button>
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)